tests/steps/application_v2_steps.py

```diff
@@ -1173,7 +1255,14 @@ def check_all_boxes(context, box_names: str = None):
     # Check that length of lists are equal, then check for set equality.
     assert len(expected_box_names) == len(
         actual_box_names
-    ), f"Expected box names array length does not match actual array length {(expected_box_names)} != {(box_response)}"
+    ), f"Expected box names array length does not match actual array length {(expected_box_names)} != {(actual_box_names)}"
     assert set(expected_box_names) == set(
         actual_box_names
     ), f"Expected box names array does not match actual array {expected_box_names} != {actual_box_names}"
+
+
+@then(
+    "I sleep for {millisecond_num} milliseconds for indexer to digest things down."
```

Comment:
How about we move this to the other sleep function: https://github.com/algorand/py-algorand-sdk/blob/c0720e9dff101cf5dc2a476ad4820cae32c469d8/tests/steps/application_v2_steps.py#L422

Reply:
yea I merged with a tricky way, hopefully it works out

Reply:
okay seems the trick works